src/components/Editor/ItemEditor.vue

```diff
@@ -510,6 +540,9 @@ export default {
   },
 
   computed: {
+    surveyModeTooltip() {
+      return this.$t("tooltip.editor.item_editor.survey_mode");
+    },
```

Comment:
no need to define this separately - can directly add the tooltip text in the template itself